docs/topics/mpp/migrating-multiplatform-project-to-14.md

```diff
@@ -116,21 +116,36 @@ Learn more about [publishing multiplatform libraries](mpp-publish-lib.md).
 
 ### Migrate to the hierarchical project structure
 
-A hierarchical project structure allows reusing code in similar targets, as well as publishing and consuming libraries with granular APIs targeting similar platforms. 
-We recommend that you switch to the hierarchical project structure in your libraries when migrating to Kotlin 1.4.0:
+A hierarchical project structure allows reusing code in similar targets, as well as publishing and consuming libraries with granular APIs targeting similar platforms. We recommend that you switch to the hierarchical project structure in your libraries when migrating to Kotlin 1.4.0:
```

Comment:
General question: do we want to keep 1.4.0 here? We have later Kotlin versions now, does it make sense to keep 1.4.0 here?

Reply:
Now, it' doesn't, it's related to all kinds of migrations from 1.3. Should I use "1.4" or "1.4.x" then?

Reply:
I would say `Kotlin 1.4.0 or higher`. We'll release 1.5.0 soon ^)